.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+on:
+  pull_request:
+  push:
+    # branches:
```

Comment:
I assume this is temporary?

Reply:
Yup. I added it in "temporarily enable branch builds" 😉 

I'll just remove it now though, having a test PR is just as straightforward. Since both the branch / tag filters are negated it makes more sense to just remove `push` entirely.